src/mainboard/seeed/beaglev/Makefile

```diff
@@ -0,0 +1,8 @@
+TARGET     = riscv64imac-unknown-none-elf
+QEMU       ?= qemu-system-riscv64
+QEMU_FLAGS += -m 1g -machine sifive_u,start-in-flash=true -nographic -device loader,addr=0x20000000,file=${IMAGE} -bios none -smp 4
```

Comment:
The machine type is still sifive_u, is this fine?

Reply:
it's a weird machine. ALl their docs reference that core.